### ISSUE TITLE
Accessibility - Persona should allow for defining the aria role

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#4401]
### Bug Report

- __Package version(s)__: 5.57
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Planner

#### Describe the issue:

We have an accessibility bug filed against us because the control type is announced by screen readers as "group" on the persona focusable element, but we are using the persona as a button and should be able to control that.

**Comment by michaelangotti** [comment]
@daweffe Since you are more familiar with this scenario, would you be willing to submit a PR? If not, can you provide a code pen repro explaining what you are asking for?
Thanks!

**Comment by KevinTCoughlin** [comment]
@daweffe [PersonaBase](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Persona/Persona.base.tsx) spreads `...divProperties` on the top-most wrapper element for Persona controls https://github.com/OfficeDev/office-ui-fabric-react/blob/40b585d6ba42d45a2302c9161612461d8ab2fca9/packages/office-ui-fabric-react/src/components/Persona/Persona.base.tsx#L121

You should be able to provide `role="button"` in your component's TSX and it will render with it.

```js
<Persona
  { ...examplePersona }
  size={ PersonaSize.size10 }
  role="button"
  onClick={ () => console.log('Persona button clicked') }
/>
```

Does that not work? If not can you please provide an example with your component so I can better see the DOM structure and problem.

Meanwhile, let me put together a Codepen to help demonstrate the above proposed solution.

**Comment by KevinTCoughlin** [comment]
@daweffe, I'm going to close this for now since this seems possible without a library change like in the example above. Please re-open if that's not the case. Thank you!
